src/opentimelineio/CORE_VERSION_MAP.cpp

```diff
@@ -15,6 +15,38 @@
 namespace opentimelineio { namespace OPENTIMELINEIO_VERSION {
 
 const label_to_schema_version_map CORE_VERSION_MAP {
+    { "0.14.0",
```

Comment:
Should we also have 0.14.1?

Reply:
Good question, @JeanChristopheMorinPerso. It would be identical to 0.14.0 right? Should we include every release from now on, or just add one when a schema changes?

Reply:
I'm under the impression we don't need 0.14.1 since it only contains bug fixes. But I referred to ask just in case. Having all versions might make it easier for users to just say "write an OTIO file using that version" without knowing which version contained schema changes. But at the same time it's not like it's really difficult to get this map and figure out pretty quickly which version is required.

Reply:
Design notes:

- for this release, I think leaving it like this is fine, since we're backporting it anyway and don't have a strategy
- I think as a project we should have a discussion.  There are two strategies that I can think of off the top of my head:
  - we base everything off the library release version:
    - every release gets an entry
    - +: its straightforward, the only thing anyone needs to know is what version of the library you have
    - -: (potential) we'll end up with a lot of entries that are the same
    - -: (potential) we conflate software release with schema changes (less obvious when things are _actually_ incompatible schema-wise
  - we create a separate "schema version":
    - +: Clear when things actually change
    - +: might be more straightforward for nontechnical end users building pipelines
    - -: adds another version number to track for folks, need to add tools for getting it and possibly recording it, etc.